README.md

```diff
@@ -1,4 +1,9 @@
-# `geoarrow-rs`
+# geoarrow-rs
+
+[![GitHub Workflow Status (CI)](https://img.shields.io/github/actions/workflow/status/geoarrow/geoarrow-rs/ci.yml?branch=main&style=for-the-badge)](https://github.com/geoarrow/geoarrow-rs/actions/workflows/ci.yml)
+[![docs.rs](https://img.shields.io/docsrs/geoarrow?style=for-the-badge&label=docs.rs)](https://docs.rs/geoarrow/latest/geoarrow/)
+[![Crates.io](https://img.shields.io/crates/v/geoarrow?style=for-the-badge)](https://crates.io/crates/geoarrow)
+![Crates.io](https://img.shields.io/crates/l/geoarrow?style=for-the-badge)
```

Comment:
```suggestion
[![GitHub Workflow Status (CI)](https://img.shields.io/github/actions/workflow/status/geoarrow/geoarrow-rs/ci.yml?branch=main)](https://github.com/geoarrow/geoarrow-rs/actions/workflows/ci.yml)
[![docs.rs](https://img.shields.io/docsrs/geoarrow?label=docs.rs)](https://docs.rs/geoarrow/latest/geoarrow/)
[![Crates.io](https://img.shields.io/crates/v/geoarrow)](https://crates.io/crates/geoarrow)
![Crates.io](https://img.shields.io/crates/l/geoarrow)
```

Personal preference but I like the original badge styles better

[![GitHub Workflow Status (CI)](https://img.shields.io/github/actions/workflow/status/geoarrow/geoarrow-rs/ci.yml?branch=main)](https://github.com/geoarrow/geoarrow-rs/actions/workflows/ci.yml)
[![docs.rs](https://img.shields.io/docsrs/geoarrow?label=docs.rs)](https://docs.rs/geoarrow/latest/geoarrow/)
[![Crates.io](https://img.shields.io/crates/v/geoarrow)](https://crates.io/crates/geoarrow)
![Crates.io](https://img.shields.io/crates/l/geoarrow)